### PR TITLE
Docs: The Insights Module

### PR DESCRIPTION
**Pull request by erondpowell**
Hey guys,

Here is the insights docs draft.
The links are not added yet and all the photos need to be totally redone.
I started realizing that last night, and stopped making/editing more pics.

Some formatting is still slightly off from the other docs. 
If you notice anything that _screams at you_ or you feel is so nuanced I may not pickup on it- let me know, please!

One key thing is Ieft out basically all the original copy from Insights.
I felt the UI was so intuitive that like core CRUD operations were just obvious.
Should add those "click the + icon to create a panel" instructions back in??

### CODE REVIEW

File: docs/app/insights.md
Comment (joselcvarela) [inline docs/app/insights.md:24]: Is this image working?
I am not seeing any inserted `image.gif` in this PR 🤔

File: docs/app/insights.md
Comment (joselcvarela) [inline docs/app/insights.md:33]: We need to wait for the video or it was not included here?

File: docs/app/insights.md
Comment (joselcvarela) [inline docs/app/insights.md:47]: Since we may want users to quickly find these images on Google you should improve the description keywords.

For example, check this better approach
```suggestion
![Edit / delete panel in Directus Insights / dashboard](../assets/insights-images/insights_dashboard_edit-and-delete-buttons.gif)
```

Do you see the relevant keywords? Basically you just need to think how you would do a search to find this image. The same applies to all the other images 👍

**Review by joselcvarela — COMMENTED** [review]
It looks awesome to me 🙌
Just need a few things to be ready IMO

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:33]: @joselcvarela 
This doc is NOT intended to be complete. It is in draft mode.

Images will need to be completely redone.
(We haven't seemed to decide on Webp vs PNG, width, etc..)

Wanted to get this up and discuss what is working and what needs improvement.

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:47]: Yes, thanks @joselcvarela

File: docs/app/insights.md
Comment (joselcvarela) [reply docs/app/insights.md:33]: When you think a PR has some work to do, you can create a Draft PR instead of a normal PR.
I am converting this to draft then 👍

File: docs/app/insights.md
Comment (benhaynes) [inline docs/app/insights.md:3]: > module in Directus

I usually capitalize our extensions (eg: `Module`), but we should decide if that's something we want to do moving forward. Also, maybe we say "Module _of_ Directus"?

File: docs/app/insights.md
Comment (benhaynes) [inline docs/app/insights.md:26]: Not sure if it was merged, but we should start using the inline icons that @Nitwel created: `+`

Comment (rijkvanzanten) [reply docs/app/insights.md:26]: Not yet, as we're blocked on figuring out final image specs before that one can be merged 👍🏻

File: docs/app/insights.md
Comment (rijkvanzanten) [reply docs/app/insights.md:3]: I'd like to keep all our "custom names" capitalized, so it'll be a Module / Interface / Display etc etc

File: docs/app/insights.md
Comment (benhaynes) [inline docs/app/insights.md:93]: > show the site's top 5 selling products

Let's be careful about being too specific/leading within use-cases. These are good examples, but we should just say "you could show your top 5 selling products"... which removes "the site's" since there's a good chance the project is not for a website. Or as easy as "you could show a website's top 5 selling products" (with "a" being less direct about managing a website).

File: docs/app/insights.md
Comment (benhaynes) [inline docs/app/insights.md:103]: Let's be consistent with bullet lists, we should either end them in a period or not. I also change this up depending on text length, but we should be the same within each list.

File: docs/app/insights.md
Comment (benhaynes) [inline docs/app/insights.md:105]: Always good to reference our proprietary config "stuff" with links to other Docs. In this case, "Display Templates" are used throughout the system, so we should have dedicated directions for how to work with this, and then link to that.

Comment (benhaynes) [reply docs/app/insights.md:105]: eg: add a "Learn more" link at the end

File: docs/app/insights.md
Comment (benhaynes) [inline docs/app/insights.md:110]: Might need to update this, @jaycammarano just updated this to handle more than numbers. 🙂

File: docs/app/insights.md
Comment (benhaynes) [inline docs/app/insights.md:122]: Again, let's update based on @jaycammarano's PR... I think certain functions just gray out based on the field.

File: docs/app/insights.md
Comment (benhaynes) [reply docs/app/insights.md:110]: Also, we're speaking with devs, but a brief overview of what aggregate functions are and can do might be nice. Or at least a `(eg: SUM, AVERAGE, etc)`.

Comment (benhaynes) [reply docs/app/insights.md:110]: Aaaand, `FIRST` and `LAST` aren't aggregate functions, so we should call those out. :)

File: docs/app/insights.md
Comment (benhaynes) [inline docs/app/insights.md:183]: Let's reference global explanations... this should link to the /Glossary page

File: docs/app/insights.md
Comment (benhaynes) [inline docs/app/insights.md:177]: Again, Filters are a pretty global thing, we might want to have a really solid explanation somewhere, and link there. We can still give contextual guidance on this page, but should link to "learn more" within the main section

File: docs/app/insights.md
Comment (benhaynes) [inline docs/app/insights.md:189]: If we do a formal video, let's make sure to create 2-3 really nice dashboards for the example. The ones in our demos have obviously fake data and should be polished up a bit.

**Review by benhaynes — CHANGES_REQUESTED** [review]
Looking good! Tried to give as much feedback as possible, I'm sure you'll need less and less guidance over time. 🙂

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:3]: Okay swag. I saw one instance in the docs where that wasn't the case and rolled with lowercase.
Def down to capitalize and had/have the impulse in my fingers to do the same.
(If I'm becoming the main tech docs geezer, I'm happy to shred them apart and touch up- once I know all the standards)

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:26]: okay sweet. "page header". noted.

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:41]: Okay, cool.
I have been trying to keep that "7th grader" vocab on indirect language.
Will turn it up a step.

@benhaynes @rijkvanzanten what else can they do?

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:189]: @benhaynes  for the first "real" PR, I'm going to cut this entirely then add it back later... But wanted to note it now.
Over the next few weeks, I'd like to help develop an "insanely swag" practice database.
... Something generic enough for beginners, robust enough for "pros" to take respectably, and a data model that covers several major use cases.... Def need some time to get my feet wet, though.

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:29]: I'm not totally sure what you mean. 
Just like add an in-page link to the panel-types listed below?

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:183]: Foreign Key is not in the glossary yet.
(But I can add it real soon)

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:177]: I expanded the filters section earlier this evening. But honestly, I started to feel the scope started ballooning like crazy and cut it with some (expanded but) basic explanations.

I'm assuming the readers have zero SQL knowledge, but decent math, actively do analytics, and are pretty fire in excel.... Is that about right for these dashboard creators?

I'd like to take the approach that _the readers know what filters are and just need a primer on how ours work_ in the main docs... Then get super-duper granular in a 1 to 4 hour tutorial series... Let me know your thoughts on that strategy!

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:110]: Yeah, I originally had one and it started to seem overwhelmingly large and like it belonged in more of a tutorial.
I will add something back in.

File: docs/app/insights.md
Comment (benhaynes) [reply docs/app/insights.md:26]: In general, we just want to re-use and reference this naming system... just helps stay consistent: https://docs.directus.io/app/overview/

File: docs/app/insights.md
Comment (benhaynes) [reply docs/app/insights.md:29]: This line references "Selecting a panel type"... but they won't know the difference between types yet. So I'm recommending adding a link here (eg `Learn more about Panel types`) to where you describe the types/options lower on this page.

File: docs/app/insights.md
Comment (benhaynes) [reply docs/app/insights.md:41]: No worries. Yeah, you can turn CRUD on/off... or use custom filters for dynamic permissions, adjust individual fields, and set validation/defaults. Just mentioning this so we say "configure permissions" instead of hinting that users just toggle them "on and off". 🙂 

https://docs.directus.io/configuration/users-roles-permissions/#configuring-permissions

File: docs/app/insights.md
Comment (benhaynes) [reply docs/app/insights.md:110]: Docs should be comprehensive in terms of what we offer (I'd even like to LIST all the functions we support here with what they do). We'll only switch to Tutorials when the topic becomes subjective or includes external software integrations.

**Comment by rijkvanzanten** [comment]
Deploy preview for this change:

https://deploy-preview-11175--directus-docs.netlify.app/app/insights/

File: docs/app/insights.md
Comment (benhaynes) [reply docs/app/insights.md:177]: Anything within the Docs "App Guide" should be geared towards anyone (non-technical too) as a baseline, but with hints and details that help out the more technical power-users and admins.

> I'd like to take the approach that the readers know what filters are and just need a primer on how ours work in the main docs... Then get super-duper granular in a 1 to 4 hour tutorial series... Let me know your thoughts on that strategy!

Pretty close! I like this, and we can include permission tutorials for a bunch of subjective use-case examples. However, I do want to have dedicated sections in the Docs that thoroughly explain working with both permissions and filters (separately).

In general, we should have ONE dedicated and comprehensive place in the Docs for each feature/component/concept. Then, we reference/link to those authoritative sections throughout the docs as needed. For example, here in Insights (context, App Guide) we would simply mention Filters (maybe with 1+ relevant examples for this context)... but would link to the main section on how to work with Filters (authoritative, Configuration).

Make sense? This is chicken and the egg, since we'll need to make sure we have all those single-source-of-truth docs written first, so that we can link to them.

File: docs/app/insights.md
Comment (jaycammarano) [reply docs/app/insights.md:122]: Yes.

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:33]: Okay, sounds good. "draft PR" noted.

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:26]: Okay. I will straight-up make flashcards to learn the Directus terms.
Will take time to get it all in my blood though.

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:29]: You're right! I'm going to link it and also add a soft-intro paragraph right here as well....

I just added "First, create a Dashboard. Then fill the Dashboard with Panels. Each Panel provides some unique type of analytics insight". LMK your thoughts. Obviously you will see it live soon, too!

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:50]: okay. noted!

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:93]: Got it. It's going to take some time and repetition to subconsciously get over the paradigm shift that this is a general purpose DB wrapper but I totally see what you mean. Thanks

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:103]: I would prefer to end them without a period, mainly because there is no direct subject in the grammar on these (and it encourages a trend towards short, simple explanations- which I think fits these "App Guide" docs. This instance is a lack of polish. But period or no period, I totally yield on whatever you want. Let me know!

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:105]: Noted! Will add.

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:110]: Wow, okay. I can do it! But listing it will be a heavy duty, long doc.... OR since there is repetition among panel types, and new panels types are coming in steadily, I may need to fundamentally restructure my approach.

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:183]: "Mirror" it?? You mean add it into the glossary section? @benhaynes

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:177]: Yeah, I get it! I mentioned to Rijk and Nils that I would love to add some dictionary table(s) DB that holds all of our terms and lets a user hover over some term and get a tooltip definition and "learn more" button. That's obviously a ways out. But I digress- "long story long", I think this approach is great. However, it will mean multiple and constant rounds of docs updates as more and more pages get added over the next several months (which I am totally happy about).

File: docs/app/insights.md
Comment (benhaynes) [reply docs/app/insights.md:110]: There are only 7-10 functions/aggregations... a short and simple bullet list with a short sentence on each is fine.

File: docs/app/insights.md
Comment (benhaynes) [reply docs/app/insights.md:183]: Yup... copy/paste the PK to create a FK section

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md:41]: Okay, sweet. I will write something like "fully configurable" and hyperlink it with that URL you just provided.

**Comment by erondpowell** [comment]
Unresolved Issues:

the length of the `-` is much longer on the other docs vs on insights and I can't see why.

`[[toc]]`
Doesn't add in `##` and `###` headers. Couldn't find this notation after multiple rounds of googling.

Should the definition of Aggregate Functions go into the Glossary or the insights doc? (I think it should go in the glossary)

Hyperlinks to other docs don't jump to `/#section-header` when testing.

Rules for hyperlinking recurring "glossary" words.

** List Panel Options > Display Template:** 
This will eventually become a full page. for now, the definition is:    *Defines how list items will be represented. Allows use of field item values as well as custom text.* ... Later on it will be hyperlinked. Will that be good enough for the time being? Or should I expand this asap?

Video delivered in iframe... Not sure how we want to handle the HTML/CSS for it.

**How to highlight menu options/buttons**
::: tip Why are some Fields grayed out?
Certain Fields cannot be used with certain aggregate functions. For example, you cannot use the average function on words, because words cannot be averaged. When something like this happens, Fields will be grayed out and un-selectable in the `field dropdown list`.

When I run git commit, it reformats the `::: tip` and `::: warning` syntax causing a UI bug! Perhaps it has to do with a plugin in VSC... but then why would it only reformat on commits?

**Comment by rijkvanzanten** [comment]
> [[toc]]
> Doesn't add in ## and ### headers. Couldn't find this notation after multiple rounds of googling.

That sounds like a VuePress configuration to me. There might be configuration flag somewhere to have it inject h2 (`##`) and h3 (`###`) headers.

> Should the definition of Aggregate Functions go into the Glossary or the insights doc? (I think it should go in the glossary)

I think in the glossary.

> Hyperlinks to other docs don't jump to /#section-header when testing.

Odd, how are you testing that exactly? Does the same problem exist on the live docs website?

> This will eventually become a full page. for now, the definition is:    Defines how list items will be represented. Allows use of field item values as well as custom text. ... Later on it will be hyperlinked. Will that be good enough for the time being? Or should I expand this asap?

I think it's fine for now. Having it is better than not having it, and we can't get too stuck on making it perfect in the first cycle. We'll have to really treat this docs upgrade as an iterative process, otherwise we'll spend an infinite amount of fine-tuning, and never go live

> Video delivered in iframe... Not sure how we want to handle the HTML/CSS for it.

Should be as simple as

```html
<video autoplay muted loop>
  <source src="https://cdn.directus.io/etc/etc/etc.mp4" type="video/mp4" />
</video>
```

> When I run git commit, it reformats the ::: tip and ::: warning syntax causing a UI bug! Perhaps it has to do with a plugin in VSC... but then why would it only reformat on commits? 

This is most likely a formatting issue then. Make sure to always have a whiteline before and after the `:::` block for it to work properly:

```
Some other text

::: tip Example!

This is the content of the tip block

:::

Some other paragraph
```

File: docs/app/insights.md
Comment (rijkvanzanten) [reply docs/app/insights.md]: > It smells to me like a new Aggregates doc may in order?

Makes sense to me. It's also noted under the API reference: https://docs.directus.io/reference/query/#aggregation-grouping, but I don't want to point less technical users there for this info 👍🏻

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md]: > > [[toc]]
> > Doesn't add in ## and ### headers. Couldn't find this notation after multiple rounds of googling. 
>
> That sounds like a VuePress configuration to me. There might be configuration flag somewhere to have it inject h2 (`##`) and h3 (`###`) headers.
> 
Okay, I haven't messed with VuePress ever, but will take some time to get to know it this week.

> > Should the definition of Aggregate Functions go into the Glossary or the insights doc? (I think it should go in the glossary)
> 
> I think in the glossary.
> 
Cool cool!

> > Hyperlinks to other docs don't jump to /#section-header when testing.
> 
> Odd, how are you testing that exactly? Does the same problem exist on the live docs website?
> 
I simply just opened the docs in dev mode and tried to click on the hyperlinks.

> > This will eventually become a full page. for now, the definition is:    Defines how list items will be represented. Allows use of field item values as well as custom text. ... Later on it will be hyperlinked. Will that be good enough for the time being? Or should I expand this asap?
> 
> I think it's fine for now. Having it is better than not having it, and we can't get too stuck on making it perfect in the first cycle. We'll have to really treat this docs upgrade as an iterative process, otherwise we'll spend an infinite amount of fine-tuning, and never go live
> 

Sweet. I'm all about it. These last few days my understanding of the scope of the docs has expanded quite a bit!

> > Video delivered in iframe... Not sure how we want to handle the HTML/CSS for it.
> 
> Should be as simple as
> 
> ```
> <video autoplay muted loop>
>   <source src="https://cdn.directus.io/etc/etc/etc.mp4" type="video/mp4" />
> </video>
> ```

Using `<iframe>` made it a tiny little icon with black padding on the sides. `<Video>` delivers it full sized, without rounded edges. Some custom CSS will need to be added. It doesn't fit to column and. (I can do that but I have taken a peek at Dircetus CSS and it will take me a while to be confident I'm matching your CSS styling nuances properly)

> > When I run git commit, it reformats the ::: tip and ::: warning syntax causing a UI bug! Perhaps it has to do with a plugin in VSC... but then why would it only reformat on commits?
> 
> This is most likely a formatting issue then. Make sure to always have a whiteline before and after the `:::` block for it to work properly:
> 
> ```
> Some other text
> 
> ::: tip Example!
> 
> This is the content of the tip block
> 
> :::
> 
> Some other paragraph
> ```

There is a blank line before and after. I will go ham on google though to try and sort this out though.

**Comment by erondpowell** [comment]
> > [[toc]]
> > Doesn't add in ## and ### headers. Couldn't find this notation after multiple rounds of googling. 
>
> That sounds like a VuePress configuration to me. There might be configuration flag somewhere to have it inject h2 (`##`) and h3 (`###`) headers.
> 
Okay, I haven't messed with VuePress ever, but will take some time to get to know it this week.

> > Should the definition of Aggregate Functions go into the Glossary or the insights doc? (I think it should go in the glossary)
> 
> I think in the glossary.
> 
Cool cool!

> > Hyperlinks to other docs don't jump to /#section-header when testing.
> 
> Odd, how are you testing that exactly? Does the same problem exist on the live docs website?
> 
I simply just opened the docs in dev mode and tried to click on the hyperlinks.

> > This will eventually become a full page. for now, the definition is:    Defines how list items will be represented. Allows use of field item values as well as custom text. ... Later on it will be hyperlinked. Will that be good enough for the time being? Or should I expand this asap?
> 
> I think it's fine for now. Having it is better than not having it, and we can't get too stuck on making it perfect in the first cycle. We'll have to really treat this docs upgrade as an iterative process, otherwise we'll spend an infinite amount of fine-tuning, and never go live
> 

Sweet. I'm all about it. These last few days my understanding of the scope of the docs has expanded quite a bit!

> > Video delivered in iframe... Not sure how we want to handle the HTML/CSS for it.
> 
> Should be as simple as
> 
> ```
> <video autoplay muted loop>
>   <source src="https://cdn.directus.io/etc/etc/etc.mp4" type="video/mp4" />
> </video>
> ```

Using `<iframe>` made it a tiny little icon with black padding on the sides. `<Video>` delivers it full sized, without rounded edges. Some custom CSS will need to be added. It doesn't fit to column and. (I can do that but I have taken a peek at Dircetus CSS and it will take me a while to be confident I'm matching your CSS styling nuances properly)

> > When I run git commit, it reformats the ::: tip and ::: warning syntax causing a UI bug! Perhaps it has to do with a plugin in VSC... but then why would it only reformat on commits?
> 
> This is most likely a formatting issue then. Make sure to always have a whiteline before and after the `:::` block for it to work properly:
> 
> ```
> Some other text
> 
> ::: tip Example!
> 
> This is the content of the tip block
> 
> :::
> 
> Some other paragraph
> ```

There is a blank line before and after. I will go ham on google though to try and sort this out though.

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md]:
```
::: tip Why are some Fields grayed out?

Certain Fields cannot be used with certain aggregate functions. For example, you cannot use the average function on Fields that store text, because text cannot be averaged. When there is a conflict between panel options, Fields will be grayed out and un-selectable in the `field dropdown list`.

:::
```

Just confirming this is accurate.

File: docs/app/insights.md
Comment (erondpowell) [reply docs/app/insights.md]: > Aaaand, `FIRST` and `LAST` aren't aggregate functions, so we should call those out. :)

Ben, here is how I addressed that:
```
::: tip First, Last, Minimum and Maximum

First, Last, Minimum and Maximum are not aggregates in the purely mathematical sense of the word.
However, they are placed in this panel option because they return one single Item.

:::
```

**Comment by erondpowell** [comment]
@benhaynes @rijkvanzanten 

Hey guys,

There is one issue left: The CSS on the `<video>` elements. I spent 10-20 minutes taking a look at the docs CSS and wasn't 100% sure what was going on, so I am opting to chat with you guys about how to address that.  I am eager to add in `video` styling and make a PR, but to save time I would love a little verbal primer around what's going on before I dive in.

Aside from that, I think this draft is ready to go live. Let me recap everything for review:
- All Directus-specific language is capitalized.
- Periods added to bullet lists.
- Image descriptions optimized.
- Steps to "How it Works" clarified.
- "Learn More about panel types" added.
- Adjusted "use-cases".
- First, Last, Min, Max are called out.
- Images are on AWS and removed from Docs.
- `[[TOC]]` I opted to redo my headers for now instead of adjust Vuepress because that is global and affects all docs.
- All `-` replaced with `–`.
- `:::` formatting is fixed.
- Hyperlink not jumping to the page header issue is local. Works fine on netlify.
- "Video Sections" removed temporarily until video is made.

TO DO LIST - Items related to Insights:
- Add inline icons
- Adding "power-user tips and tricks" kept in mind. Will add more cyclically.
- Filter options Doc
- Display Templates Doc
- "If we do a formal video, let's make sure to create 2-3 really nice dashboards".
- Aggregate Functions and Foreign Key terms will go into the Glossary! (on a separate PR, right?)

**Comment by benhaynes** [comment]
For the video styling, I think we'd just want to inherit the img styling (margins, rounded corners, etc). The controls will be in the video tag itself: autoplay, start-muted, loop, etc...

@rijkvanzanten can help get the template down.

**Comment by rijkvanzanten** [comment]
For both images and videos, we could do something like:

```css
border-radius: 8px;
max-width: 100%;
filter: drop-shadow(0 15px 11px rgb(0 0 0 / 0.2));
```

CleanShot 2022-01-26 at 09 25 24@2x

@benhaynes could you fiddle with the exact shadow?

**Comment by benhaynes** [comment]
Let's use this.
```
box-shadow: 0px 10px 20px 0px rgba(23,41,64,0.2), 0px 4px 8px 0px rgba(23,41,64,0.2);
```

**Comment by rijkvanzanten** [comment]
Screenshot of the above box shadow:

CleanShot 2022-01-26 at 09 49 10@2x

**Comment by erondpowell** [comment]
> Screenshot of the above box shadow:
> 
> CleanShot 2022-01-26 at 09 49 10@2x

Sweet. You guys want me to make a new PR for this? Or have somebody else do it? It might take me a few hrs because I'm not familiar with that `.styl` syntax and need to also find where the docs image CSS lives.

**Comment by rijkvanzanten** [comment]
> Sweet. You guys want me to make a new PR for this? Or have somebody else do it? It might take me a few hrs because I'm not familiar with that `.styl` syntax and need to also find where the docs image CSS lives.

We'll take care of it 👍🏻

**Comment by erondpowell** [comment]
> > Sweet. You guys want me to make a new PR for this? Or have somebody else do it? It might take me a few hrs because I'm not familiar with that `.styl` syntax and need to also find where the docs image CSS lives.
> 
> We'll take care of it 👍🏻

please, kindly tag me on the commit so I can see how you did it.

**Comment by rijkvanzanten** [comment]
@erondpowell see https://github.com/directus/directus/pull/11175/commits/66c1ed4958747f8e9bce69853776484dd584e8f8

@benhaynes there's one image in the docs currently that looks a little funky with the default shadow. Maybe we can replace that with something that would work in this new standard visual style?

CleanShot 2022-01-26 at 14 43 06@2x

**Comment by benhaynes** [comment]
Instead, could we just add a `.no-shadow` class when we don't want that? @rijkvanzanten

**Comment by rijkvanzanten** [comment]
@benhaynes Yeah, just slightly more hassle. Markdown doesn't do image classes, so we'd have to use an `` HTML tag rather than the normal `Alt text` markdown variant.

**Comment by erondpowell** [comment]
@benhaynes @rijkvanzanten Seems highly done, guys!
Thx for the training wheels on this one (and a few more to come I'm sure)!

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**
